src/components/DocSearch/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { useRef, useEffect } from 'react';
+import './DocSearch.css';
+
+export default function DocSearch({ appId, indexName, apiKey }) {
+  const docsearchRef = useRef(null);
+  useEffect(() => {
+    if (!docsearchRef.current) {
+      return;
+    }
+    import('docsearch.js').then(({ default: docsearch }) => {
+      docsearch({
+        appId,
+        apiKey,
+        indexName,
+        inputSelector: `#${docsearchRef.current.id}`,
+        debug: true, // Set debug to true if you want to inspect the dropdown
```

Comment:
Since the input is not styled, what about leveraging `enhancedSearchInput`?

Reply:
enhancedSearchInput is broken in the latest versions, we'll do some skinning with ui-lib